src/app/store-stats/ClarityCharacterStat.tsx

```diff
@@ -0,0 +1,247 @@
+      />,
+      <StatTableRow
+        key="DamageResistance"
+        name={t('Stats.DamageResistance')}
+        cooldowns={clarityStatData.DamageResistance}
+        tier={tier}
+        unit={t('Stats.Percentage')}
+      />,
+      <StatTableRow
+        key="FlinchResistance"
+        name={t('Stats.FlinchResistance')}
+        cooldowns={clarityStatData.FlinchResistance}
+        tier={tier}
+        unit={t('Stats.Percentage')}
+      />
+    );
+  }
+
+  if (intrinsicCooldowns.length + abilityCooldowns.length === 0) {
+    return null;
+  }
+
+  return (
+    <Tooltip.Section className={styles.communityInsightSection}>
+      <h3>{t('MovePopup.CommunityData')}</h3>
+      <table>
+        <thead>
+          <tr>
+            <th />
+            {tier - 1 >= 0 && (
+              <>
+                <th>{t('LoadoutBuilder.TierNumber', { tier: tier - 1 })}</th>
+                <th />
+              </>
+            )}
+            <th className={styles.currentColumn}>{t('LoadoutBuilder.TierNumber', { tier })}</th>
+            <th />
+            {tier + 1 <= 10 && (
+              <>
+                <th>{t('LoadoutBuilder.TierNumber', { tier: tier + 1 })}</th>
+                <th />
+              </>
+            )}
+          </tr>
+        </thead>
+        <tbody>
+          {abilityCooldowns
+            .sort((a, b) => a.cooldowns[tier] - b.cooldowns[tier])
+            .map(({ cooldowns, item, overrides }) => (
+              <StatTableRow
+                key={item.hash}
+                name={item.displayProperties.name}
+                icon={item.displayProperties.icon}
+                cooldowns={cooldowns}
+                tier={tier}
+                overrides={overrides}
+                unit={t('Stats.Second')}
+              />
+            ))}
+          {intrinsicCooldowns}
+        </tbody>
+      </table>
+    </Tooltip.Section>
+  );
+}
+
+function StatTableRow({
+  name,
+  icon,
+  cooldowns,
+  tier,
+  unit,
+  overrides = [],
+}: {
+  name: string;
+  icon?: string;
+  unit: string;
+  tier: number;
+  cooldowns: number[];
+  overrides?: DestinyInventoryItemDefinition[];
+}) {
+  const unitEl = <td className={styles.unit}>{unit}</td>;
+
+  return (
+    <tr>
+      <th>
+        <span>
+          {icon && <BungieImage src={icon} height={16} width={16} />}
+          {name}
+        </span>
+        {overrides.map((o) => (
+          <span key={o.hash} className={styles.override}>
+            {' '}
+            +{' '}
+            {o.displayProperties.icon && (
+              <BungieImage src={o.displayProperties.icon} height={12} width={12} />
+            )}
+            {o.displayProperties.name}
+          </span>
+        ))}
+      </th>
+      {tier - 1 >= 0 && (
+        <>
+          <td>{cooldowns[tier - 1].toLocaleString()}</td>
+          {unitEl}
+        </>
+      )}
+      <td className={styles.currentColumn}>{cooldowns[tier].toLocaleString()}</td>
+      {unitEl}
```

Comment:
The table might be easier to scan if the units also get the current column styling.

![dim-stat-tooltip-unit-color](https://github.com/DestinyItemManager/DIM/assets/17512262/46efe2f0-20a9-4194-bc0d-aae117514874)